hoverxref/_static/js/hoverxref.js_t

```diff
@@ -60,13 +60,19 @@ function reLoadSphinxTabs() {
     };
 };
 
-function getEmbedURL(project, version, doc, docpath, section) {
-    var params = {
-        'project': project,
-        'version': version,
-        'doc': doc,
-        'path': docpath,
-        'section': section,
+function getEmbedURL(project, version, doc, docpath, section, url) {
+    if (url) {
```

Comment:
If we can detect there is data coming from Intersphinx, I wonder if we could note that in the embed UI? I wanted a `Sphinx:` or similar as a context clue about the content when I used the PR build.

Reply:
We could add an extra class, e.g. `intersphinx`, to differentiate these from other regular hoverxref links and do something specific in the frontend for them. I'll open an issue to work on this from the FE as an improvement. For now, I'm going to add the extra class to these nodes.

Reply:
I just realized that Sphinx uses `internal` and `external` classes to differentiate this. We may want to rely on that directly.

Reply:
Cool. I just think it's useful to know where the page is coming from, so maybe just a larger feature later that adds breadcrumbs or something outside the embed frame to give more context to the document.